src/tests/Favorites.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import {render} from '@testing-library/react'
+import Favorites from '../pages/Favorites'
+
+
+describe('Favorites', () => {
+    test('renders the Favorites component', () => {
+      render(<Favorites />);
```

Comment:
you can try TDD on this one, what would you expect to appear here. 
No changes needed for now.